tests/integration/targets/gem/tasks/main.yml

```diff
@@ -178,3 +178,44 @@
     that:
       - install_gem_result is changed
       - gem_search.files | length == 0
+
+# Custom directory for executables (--bindir)
+- name: Install gem with custom bindir
+  gem:
+    name: gist
+    state: present
+    bindir: "{{ output_dir }}/custom_bindir"
+    user_install: no  # Avoid conflicts between --install-dir and --user-install when running as root on CentOS / Fedora / RHEL
+  register: install_gem_result
+
+- name: Get stats of gem executable
+  stat:
+    path: "{{ output_dir }}/custom_bindir/gist"
+  register: gem_bindir_stat
+
+- name: Ensure gem executable was installed in custom directory
+  assert:
+    that:
+      - install_gem_result is changed
+        # openSUSE Leap adds `--format-executable` by default which makes bindir unusable
```

Comment:
This can (and should) be solved by applying `--norc` for install and uninstall actions however this option was added in rubygems 2.5.2 and was not available before.

Reply:
If I understood you correctly, this can be fixed by adding a `command` task before which checks rubygems version and only tests when it's >= 2.5.2, right?